src/Umbraco.Web/Models/ContentEditing/MemberSave.cs

```diff
@@ -27,16 +31,27 @@ public class MemberSave : ContentBaseSave<IMember>
         [DataMember(Name = "memberGroups")]
         public IEnumerable<string> Groups { get; set; }
 
-        [DataMember(Name = "comments")]
-        public string Comments { get; set; }
-
-        [DataMember(Name = "isLockedOut")]
-        public bool IsLockedOut { get; set; }
-
-        [DataMember(Name = "isApproved")]
-        public bool IsApproved { get; set; }
-        
-
-        // TODO: Need to add question / answer support
+        /// <summary>
+        /// Returns the value from the Comments property
+        /// </summary>
+        public string Comments => GetPropertyValue<string>(Constants.Conventions.Member.Comments);
+
+        /// <summary>
+        /// Returns the value from the IsLockedOut property
+        /// </summary>
+        public bool IsLockedOut => GetPropertyValue<bool>(Constants.Conventions.Member.IsLockedOut);
+
+        /// <summary>
+        /// Returns the value from the IsApproved property
+        /// </summary>
+        public bool IsApproved => GetPropertyValue<bool>(Constants.Conventions.Member.IsApproved);
```

Comment:
These are calculated each time. Could just be a simple getter and set the values in the default constructor 🙂 

Something like

```
...
        public MemberSave()
        {
            Comments = GetPropertyValue<string>(Constants.Conventions.Member.Comments);
            IsLockedOut = GetPropertyValue<bool>(Constants.Conventions.Member.IsLockedOut);
            IsApproved = GetPropertyValue<bool>(Constants.Conventions.Member.IsApproved);
        }
...
        /// <summary>
        /// Returns the value from the Comments property
        /// </summary>
        public string Comments { get; }

        /// <summary>
        /// Returns the value from the IsLockedOut property
        /// </summary>
        public bool IsLockedOut { get; }

        /// <summary>
        /// Returns the value from the IsApproved property
        /// </summary>
        public bool IsApproved { get; }
...
```

Reply:
We can't set the values in the ctor because the `Properties` value isn't set in the constructor, it is a get/set property which these values rely on. I was going to just have a lazy field value set to prevent recalculation but then I stopped caring :P